contributors/join-us/staff-recruiting/README.md

```diff
@@ -4,21 +4,60 @@ description: 1% complete.
 
 # Staff Recruiting
 
-In order to continually increase what we offer to customers we need to continually source, assess, hire and onboard new staff members. The below outlines the process for recruiting new staff into the company. 
+In order to continually increase what we offer to customers we need to continually source, assess, hire and onboard new staff members with [high standards](https://handbook.mattermost.com/contributors/onboarding/staff-onboarding-guide#leadership-principles). The below outlines the process for recruiting new staff into the company.  
+
+## How to Open a New Role 
+Hiring Managers should confirm with your Team's MLT that your new headcount requisition has been approved. If not, your MLT team member should work with the Director of HR and VP Finance and Operations to gain approval. 
 
 ## Role Description 
 
-Role descriptions should have clear responsibilities and requirements for each role that a recruiter can use to filter candidates and interviews can use to assess the candidate's fit for the role. 
+Once you have confirmation that your new headcount has been approved, you’re ready to begin the recruitment process. Complete the [Job Description portion](https://docs.google.com/document/d/1rpTI2NKu4H_781vpx_eD9Fku7go4gGDs1P7bJAMQvAU/edit#) and share with your Recruiter, who will schedule time to review the remainder of the in-take template. 
+
+Your Recruiter will be based on the role and team. This may differ depending on Recruiting Team bandwidth, and will be communicated on a case-by-case basis. 
```

Comment:
```suggestion
Your Recruiter is assigned based on the role and team. This may differ depending on Recruiting Team bandwidth, and will be communicated on a case-by-case basis. 
```